src/providers/Project/projectDefault.ts

```diff
@@ -165,7 +165,7 @@ export function createLocalProject(
       draftCode: script,
       deployedCode: "",
       deployedContracts: [],
-      state: ""
+      state: "{}"
```

Comment:
Why do we need a empty object string here?

Reply:
I did this as a matter of convenience. For example, I use 'Object.keys' to iterate over the state object and generate the the storage items based on the result of Object.keys. By having an empty object I get zero results and if the user has no items in storage nothing is rendered. However, just as well I can set to this an empty string at the expense of adding more logic/checks when generating the storage items. Seemed like the most elegant solution was to assume this will always be an object (empty at first). 

Please let me know your preference here. If an empty string is the right way to go I can refactor how this is all handled.

Reply:
Sure, you can use empty object to iterate with `Object.keys`, but you have a `string`, representing empty object here :)
or you `JSON.parse` it somewhere down the line?

Reply:
Ok, marking this one resolved after chatting with Max. In this case, the string that represents an empty object here is indeed used by JSON.parse down the line, and by doing so we are also removing the need for various other conditional checks elsewhere in the code.